src/v2/view-builder/views/webauthn/ChallengeWebauthnView.js

```diff
@@ -115,8 +148,93 @@ const Body = BaseForm.extend({
   }
 });
 
+const Footer = AuthenticatorFooter.extend({
+  props: {
```

Comment:
this is a view, I don't think we should add `props` to it, those belong to a model.